market/database/backends.py

```diff
@@ -465,6 +477,50 @@ def get_latest_sequence_number(self):
         db_result = self.execute(db_query).fetchone()
         return db_result[0] if db_result is not None else 0
 
+    def check_add_genesis_block(self, public_key_benefactor, public_key_beneficiary):
+        """
+        Persist the genesis block if there are no blocks yet in the blockchain.
+        :param public_key_benefactor: The public key of the benefactor
+        :param public_key_beneficiary The public key of the beneficiary
+        """
+        db_query = u"SELECT COUNT(*) FROM multi_chain"
+        db_result = self.execute(db_query).fetchone()
+
+        if db_result[0] == 0:
+            insert_time = int(time.time())
+            # Hash the block
+            packet = encode(
```

Comment:
Is it possible to actually create a `DatabaseBlock' that is the genesis block? And make a seperate function to generate the block and return it. Then you can just do `BlockChain.add_block(genesis)' right?